packages/now-cli/test/integration.js

```diff
@@ -493,6 +493,35 @@ test('list the payment methods', async t => {
   t.true(stdout.startsWith(`> 0 cards found under ${contextName}`));
 });
 
+test.only('domains inspect', async t => {
+  const domainName = `inspect-${contextName}.org`;
+
+  const addRes = await execa(
+    binaryPath,
+    [`domains`, `add`, domainName, ...defaultArgs],
+    { reject: false }
+  );
+  t.is(addRes.code, 0);
+
+  const { stderr, code } = await execa(
+    binaryPath,
+    ['domains', 'inspect', domainName, ...defaultArgs],
+    {
+      reject: false,
+    }
+  );
+
+  const rmRes = await execa(
+    binaryPath,
+    [`domains`, `rm`, domainName, ...defaultArgs],
+    { reject: false, input: 'y' }
+  );
+  t.is(rmRes.code, 0);
+
+  t.is(code, 0);
+  t.true(!stderr.includes(`Renewal Price`));
```

Comment:
Shouldn't this be looking at stdout to check if Renewal Price is found?

Reply:
@styfle Renewal price is only for domains that were bought with us - the domain in this test was not

Reply:
Let me rephrase my question: why are you using `stderr` instead of `stdout`?

Is that how `now domains inspect` is supposed to work? That seems wrong because it's not an error.